lib/src/pages/credits.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/material.dart";
+
+import "package:ramaz/widgets.dart";
+import "package:ramaz/data.dart";
+
+import "drawer.dart";
+
+/// Creates the Credits Page by iterating through each ContributorCard.
+class CreditsPage extends StatelessWidget {
+	/// Builds the Credits page.
+	const CreditsPage();
+
+	@override 
+	Widget build (BuildContext context) => ResponsiveScaffold(
+		drawer: const NavigationDrawer(),
+		appBar: AppBar(title: const Text ("Credits")),
+		bodyBuilder: (_) => ListView(
+			children: [
+				for (final Contributor contributor in Contributor.contributors)
+					ContributorCard(contributor)
+			]
+		)
+	);
+}
+/// A class that creates each individual Contributor's Card.
+/// It creates a Row that holds the Image, and then a Column that
+/// holds the rest of the information.
+class ContributorCard extends StatelessWidget{
+	/// contributor is a variable that must be a Contributor.
+	final Contributor contributor;
+	/// ContributorCard requires a Contributor to be passed.
+	const ContributorCard(this.contributor);
```

Comment:
```suggestion
	final Contributor contributor;
	
	/// Creates a widget to represent a [Contributor].
	const ContributorCard(this.contributor);
```

1. Leave a blank line between properties
2. The docs will say what's required or not, so you don't have to. You can just say how it will be used.